web/app/models.py

```diff
@@ -226,7 +227,7 @@ def __str__(self):
 class TournamentPlayer(db.Model):
```

Comment:
I'm not familiar enough with in-person tournaments to know: how does a day-of signup work for aga tournament reporting purposes, anyway? is the tournament data submission so slow that an AGA id has been allocated by then, and somebody manually lines up the names?